internal/cmd/clone/clone.go

```diff
@@ -19,14 +43,32 @@ func (o *options) AddFlags(cmd *cobra.Command) {
 		"",
 		"specify branch to check out",
 	)
+	cmd.Flags().VarP(
+		&o.expectedRootKeys,
+		"root-key-path",
```

Comment:
I'd make this just root-key, because it may well be a gpg fingerprint or a fulcio identity.

Reply:
Fixed